openapi.yaml

```diff
@@ -373,18 +399,58 @@ components:
                           type: string
                           description: Formatted value of keyword
                           example: '933012347'
+    DocumentTypeResource:
+      properties:
+        id:
+          type: string
+          description: The unique identifier of a document type
+        type:
+          type: string
+          enum:
+            - document-types
+        links:
+          $ref: '#/components/schemas/SelfLink'
+        attributes:
+          type: object
+          properties:
+            name:
+              type: string
+              description: Name of the document type
+              example: 'PAR PAR Form'
+            documentTypeGroupId:
```

Comment:
After this request the user has the documentTypeName, documentTypeId and documentTypeGroupId. Is there a documentTypeGroup name and would that also be useful?

Reply:
Not sure. We can revisit and add it in the future if there is a need.